src/app/features/conditions/edit-conditions/conditions.component.ts

```diff
@@ -36,6 +37,8 @@ export class ConditionsComponent extends SingleRowEditorComponent<Conditions> {
   public readonly CONDITION_VALUE_2_TOOLTIPS = CONDITION_VALUE_2_TOOLTIPS;
   public readonly CONDITION_VALUE_3_TOOLTIPS = CONDITION_VALUE_3_TOOLTIPS;
 
+  public readonly QUEST_STATE = QUEST_STATE;
```

Comment:
@Kitzunu  2) here we simply import the flag in order to make it available for the component's view (HTML)